.cspell.json

```diff
@@ -16,6 +16,9 @@
     "assets/architecture*.xml"
   ],
   "words": [
```

Comment:
I suggest we add items to arrays like this rather on top to produce better diffs.